cmd/destroy/destroy.go

```diff
@@ -95,6 +97,20 @@ func Destroy(ctx context.Context) *cobra.Command {
 		Args:  utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#destroy"),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if options.ManifestPath != "" {
+				// we need to store the original manifest path as relative path before switching the cwd
+				options.OriginalManifestPath = options.ManifestPath
+				if filepath.IsAbs(options.ManifestPath) {
+					cwd, err := os.Getwd()
+					if err != nil {
+						return err
+					}
+					relativeOriginalManifestPath, err := filepath.Rel(cwd, options.ManifestPath)
+					if err != nil {
+						return err
+					}
+					options.OriginalManifestPath = relativeOriginalManifestPath
+				}
```

Comment:
This logic seems similar in the 3 places where we have to use it, maybe we can extract it to a function and do a unit test?

Reply:
yes, i was thinking that. The change of cwd is the same at deploy destroy and up.... i have doubts on where to create this logic 👀  at the model package under the manifest utils?. what variates is the Options Struct... but we can "unify" a lot of logic...

Reply:
The logic per se is not related to the manifest. Just with paths, don't know if we have some specific package, but `path.go` file sounds good to me to store this logic. At the end, this is checking that, if a path is absolute, get the relative one with another one (I wouldn't use `os.Getwd()` directly in the function, but passing its value as a parameter)

Reply:
@ifbyol @jLopezbarb i've uploaded some changes regarding this common logic.... i'm not sure if this is the best aproach. let me know if is ok and i will add the test or refactor again :D